packages/core/src/Button/index.tsx

```diff
@@ -497,3 +498,174 @@ export const IconButton = React.forwardRef<BaseElement, IconButtonProps>(
     )
   }
 )
+
+/**
+ * IconTextButton
+ *
+ * Always has a primary icon and a secondary text
+ *
+ */
+
+export const NativeIconTextButton = styled.button<{
+  readonly icon: IconType
+  readonly visibleFocus: boolean
+}>`
+  white-space: nowrap;
+  max-width: 100%;
+  height: ${componentSize.small};
+  outline: none;
+  &::-moz-focus-inner {
+    border: 0;
+  }
+  border: none;
+  border-radius: ${shape.radius.small};
+  cursor: pointer;
+  user-select: none;
+  transition: all 200ms;
+  padding: 0 ${spacing.large} 0 0;
+  ${({ visibleFocus, theme }) => {
+    return css`
+      color: ${theme.color.text04()};
+      fill: ${theme.color.text04()};
+      background-color: transparent;
+
+      &:hover {
+        color: ${theme.color.text03()};
+        background-color: ${theme.color.element11(opacity[16])};
+        ${IconContainer} {
+          background-color: ${theme.color.textPrimary()};
+        }
+      }
+      &:focus {
+        ${visibleFocus
+          ? css`
+              color: ${theme.color.text04()};
+              background-color: ${theme.color.element11(opacity[16])};
+            `
+          : undefined};
+      }
+      &:active {
+        box-shadow: 0 0 0 4px ${theme.color.elementPrimary(opacity[24])};
+        background-color: ${theme.color.element11(opacity[24])};
+      }
+      &:disabled {
+        opacity: ${opacity[48]};
+        cursor: default;
+        box-shadow: none;
+        &:hover {
+          ${IconContainer} {
+            background-color: ${theme.color.elementPrimary()};
+          }
+        }
+      }
+    `
+  }}
+`
+const IconContainer = styled(Icon)`
+  ${({ theme }) => {
+    return css`
+      height: ${componentSize.small};
+      width: ${componentSize.small};
+      color: ${theme.color.text00()};
+      background-color: ${theme.color.elementPrimary()};
+      border-radius: 2px;
```

Comment:
You should use this instead of 2px. `border-radius: ${shape.radius.small};`

Reply:
Done.